### PR TITLE
Correct sigma computation in voxelDescriptors

### PR DESCRIPTION
**Pull request by hawk31**

### CODE REVIEW

File: htmd/molecule/voxeldescriptors.py [hunk 1]
```diff
@@ -149,34 +149,41 @@ def _findDonors(mol, bonds):
 
 
 def _getSigmas(mol):
-    """ Returns the VDW radii of each atom in the molecule
+    # Sigmas taken from VMD vdW radii.
```
Comment (stefdoerr) [inline htmd/molecule/voxeldescriptors.py]: Add a link in the comment to where you got them from

File: htmd/molecule/voxeldescriptors.py [hunk 2]
```diff
@@ -149,34 +149,41 @@ def _findDonors(mol, bonds):
...
+     'N':   1.55,
+     'NA':  1.55,
+     'OA':  1.52,
+     'P':   1.8,
+     'S':   1.8,
+     'SA':  1.8,
+     'Zn':  1.39,
+     'Se':  1.9}
 
-    Parameters
```
Comment (stefdoerr) [inline htmd/molecule/voxeldescriptors.py]: Why did you remove the docs?

File: htmd/molecule/voxeldescriptors.py (hunk 1 above)
Comment (j3mdamas) [inline htmd/molecule/voxeldescriptors.py]: I don't know if we are going for chemical correctness here, but lennard-jones sigmas and van der waals radius are not the same thing

**Comment by hawk31** [comment]
Okay, I'll make this pull request more complete tomorrow. You can close it for the moment.
